pandas/tests/io/test_feather.py

```diff
@@ -83,7 +83,9 @@ def test_basic(self):
         if pyarrow_version >= LooseVersion("0.16.1.dev"):
             df["periods"] = pd.period_range("2013", freq="M", periods=3)
             df["timedeltas"] = pd.timedelta_range("1 day", periods=3)
-            df["intervals"] = pd.interval_range(0, 3, 3)
+            # TODO temporary disable due to regression in pyarrow 0.17.1
```

Comment:
Is there a condition when this can be restored?  Eg “When X version Y is released”

Reply:
It depends on when it will be fixed in pyarrow, so difficult to say for sure at this moment

Reply:
ok thanks.  LGTM pending green